samples/core-projects/libraries/README.md

```diff
@@ -0,0 +1,83 @@
+# .NET Core Library Samples
+
+**Note:** These are compatibile with tooling that is still in active development.  Details are subject to change over time.
+
+These samples are buildable projects whose source are used for code snippets in [the guide for writing cross-platform libraries](../../../docs/libraries/libraries-with-cli.md).  They can be built and run using the .NET Core RC1 toolchain.
+
+They are small and intended to demonstrate how to target and build NuGet packages for different targets.
```

Comment:
Should this sentence be part of the preceding sentence? It seems to be missing context with it being by itself.